scss/_patterns_navigation.scss

```diff
@@ -651,21 +657,214 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  @keyframes vf-nav-fade-in {
+    from {
+      opacity: 0;
+    }
+    to {
+      opacity: 0.5;
+    }
+  }
+
+  $sliding-nav-animation-settings: map-get($animation-duration, brisk) ease;
+
+  .p-navigation--sliding.has-menu-open {
+    height: 100vh;
+    overflow-y: hidden;
+
+    @media (min-width: $breakpoint-navigation-threshold) {
+      height: auto;
+      overflow-y: visible;
+    }
+
+    .p-navigation__banner .p-navigation__items {
+      display: flex;
+      flex-wrap: nowrap;
+
+      @media (min-width: $breakpoint-navigation-threshold) {
+        display: none;
+      }
+    }
+    .p-navigation__nav {
+      display: block;
+      height: 100vh;
+      overflow-x: hidden;
+
+      .p-navigation__items {
+        display: block;
+      }
+
+      @media (min-width: $breakpoint-navigation-threshold) {
+        display: flex;
+        height: auto;
+        overflow-x: visible;
+
+        .p-navigation__items {
+          display: flex;
+        }
+      }
+    }
+
+    .p-navigation__items:not(:first-child)::before {
+      content: none;
+    }
+  }
+
+  /* stylelint-disable max-nesting-depth */
+  .p-navigation--sliding .p-navigation__items {
+    transition: transform $sliding-nav-animation-settings;
+
+    &.is-active {
+      transform: translateX(-25vw);
+      @media (min-width: $breakpoint-navigation-threshold) {
+        transform: none;
+      }
+      .p-navigation__dropdown {
+        transform: translateX(25vw);
+        @media (min-width: $breakpoint-navigation-threshold) {
+          transform: none;
+        }
+        &[aria-hidden='false'] {
+          transform: translateX(-75vw);
+          @media (min-width: $breakpoint-navigation-threshold) {
+            transform: none;
+          }
+          &.is-active {
+            transform: translateX(-100vw);
+            @media (min-width: $breakpoint-navigation-threshold) {
```

Comment:
I don't necessarily like having many little media queries that all practically do the same. But at least they are right next to the thing they are overriding.

Because of how different mobile and desktop navigations are it almost seems like it should be two separate blocks of style.

Anyway, let's leave it for now I think, but there could be some room of improvement once we make this a default or something…

Reply:
I don't know I think it makes sense for the component to render differently on different screen sizes. That way we keep the state on resizes and don't have to hack it with JS

Reply:
Sure, I don't mean to remove the media queries at all. I just wonder if they could be "compressed" a bit.

Like here we have multiple ones that do `transform: none` on the same element. 

It almost seems like single:
`.p-navigation--sliding .p-navigation__items { transform: none }` would do similar (but because of nesting and specificity I guess it would need `!important` or something.

On the other hand, it almost seems like it would be a good place to use `max-width` media query, to target small screens only. But there are other issues with that, so…

Anyway, I'm fine with leaving it as is for now, because there is probably no easy and clean way to fix it.